[2024-09-27T13:03:31.068+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-27T13:03:31.103+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: find_activity.read_activity_from_file scheduled__2024-09-26T00:00:00+00:00 [queued]>
[2024-09-27T13:03:31.124+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: find_activity.read_activity_from_file scheduled__2024-09-26T00:00:00+00:00 [queued]>
[2024-09-27T13:03:31.126+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-09-27T13:03:31.162+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): read_activity_from_file> on 2024-09-26 00:00:00+00:00
[2024-09-27T13:03:31.174+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=91) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-27T13:03:31.181+0000] {standard_task_runner.py:72} INFO - Started process 93 to run task
[2024-09-27T13:03:31.180+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'find_activity', 'read_activity_from_file', 'scheduled__2024-09-26T00:00:00+00:00', '--job-id', '44', '--raw', '--subdir', 'DAGS_FOLDER/find_activity.py', '--cfg-path', '/tmp/tmpu203itah']
[2024-09-27T13:03:31.184+0000] {standard_task_runner.py:105} INFO - Job 44: Subtask read_activity_from_file
[2024-09-27T13:03:31.301+0000] {task_command.py:467} INFO - Running <TaskInstance: find_activity.read_activity_from_file scheduled__2024-09-26T00:00:00+00:00 [running]> on host a079404de250
[2024-09-27T13:03:31.537+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='find_activity' AIRFLOW_CTX_TASK_ID='read_activity_from_file' AIRFLOW_CTX_EXECUTION_DATE='2024-09-26T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-26T00:00:00+00:00'
[2024-09-27T13:03:31.540+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-27T13:03:31.546+0000] {logging_mixin.py:190} INFO - Today you will: Explore a park you have never been to before
Today you will: Volunteer and help out at a senior center
[2024-09-27T13:03:31.548+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-27T13:03:31.569+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-27T13:03:31.571+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=find_activity, task_id=read_activity_from_file, run_id=scheduled__2024-09-26T00:00:00+00:00, execution_date=20240926T000000, start_date=20240927T130331, end_date=20240927T130331
[2024-09-27T13:03:31.644+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-27T13:03:31.680+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-27T13:03:31.689+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
